docker/ci/build.ps1

```diff
@@ -19,14 +32,14 @@ function Build-Container($Name, $Repository = '127.0.0.1')
 {
   Push-Location (Join-Path (Get-CurrentDirectory) '..')
   bundle exec puppet-docker local-lint $Name
-  bundle exec puppet-docker build $Name --no-cache --repository $Repository --build-arg namespace=$Repository
+  bundle exec puppet-docker build $Name --no-cache --repository $Repository --version $Version --no-latest --build-arg namespace=$Repository
```

Comment:
Missing  $Version parameter in function def

Reply:
jinx ;0

Reply:
I don't know what's wrong with me today!